<ui version="4.0" >
 <class>Dialog_TemplateMatchingSegPara</class>
 <widget class="QDialog" name="Dialog_TemplateMatchingSegPara" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Template matching segmentation parameters</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>11</y>
     <width>260</width>
     <height>503</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" >
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Channel</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="spinBox_Channel" >
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Kernel (template)</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="comboBox_Kernel" >
          <item>
           <property name="text" >
            <string>Gaussian</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" >
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Dimension 
(radius, # pixels)</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>x</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QSpinBox" name="spinBox_szx" >
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="value" >
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>y</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QSpinBox" name="spinBox_szy" >
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="value" >
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>z</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QSpinBox" name="spinBox_szz" >
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="value" >
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="1" >
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Standard Deviation</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="_2" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>x</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>y</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_9" >
              <property name="text" >
               <string>z</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QDoubleSpinBox" name="doubleSpinBox_stdz" >
              <property name="minimum" >
               <double>0.500000000000000</double>
              </property>
              <property name="value" >
               <double>1.500000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QDoubleSpinBox" name="doubleSpinBox_stdy" >
              <property name="minimum" >
               <double>0.500000000000000</double>
              </property>
              <property name="value" >
               <double>3.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QDoubleSpinBox" name="doubleSpinBox_stdx" >
              <property name="minimum" >
               <double>0.500000000000000</double>
              </property>
              <property name="value" >
               <double>3.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Thresholds</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>Pixel value >=</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QDoubleSpinBox" name="doubleSpinBox_pixelval" >
           <property name="maximum" >
            <double>65535.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>10.000000000000000</double>
           </property>
           <property name="value" >
            <double>50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_12" >
           <property name="text" >
            <string>Local region pixel 
mean value >=</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="doubleSpinBox_rgnval" >
           <property name="maximum" >
            <double>65535.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>10.000000000000000</double>
           </property>
           <property name="value" >
            <double>40.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QDoubleSpinBox" name="doubleSpinBox_corrcoef" >
           <property name="maximum" >
            <double>1.000000000000000</double>
           </property>
           <property name="value" >
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_13" >
           <property name="text" >
            <string>Correlation >=</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Other parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_14" >
         <property name="text" >
          <string>region merge radius</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QDoubleSpinBox" name="doubleSpinBox_rgn_merge_radius" >
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>10.000000000000000</double>
         </property>
         <property name="value" >
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QPushButton" name="pushButton_cancel" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ok" >
        <property name="text" >
         <string>OK</string>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
